package.json

```diff
@@ -1,64 +1,79 @@
 {
-  "name": "singularity-directory-webapp-react",
+  "name": "react-next-boilerplate",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "test": "echo 'TODO: we need to add some tests on this project as soon as possible!'",
-    "eslint-staged": "files=`git diff --staged --diff-filter=AM --name-only | grep .js$ | tr '\n' ' '`; if [ -n \"$files\" ]; then eslint $files; fi;",
-    "eslint": "eslint .",
     "dev": "node server.js",
     "build": "next build src",
     "start": "NODE_ENV=production node server.js",
-    "storybook": "start-storybook -p 9009 -c storybook",
-    "precommit": "npm run eslint-staged",
-    "prepush": "npm run eslint && npm run test"
+    "storybook": "start-storybook -p 6006",
+    "eslint": "eslint ./src",
+    "stylelint": "stylelint ./src/**/*.css",
+    "lint": "yarn eslint && yarn stylelint",
+    "build-storybook": "build-storybook"
   },
-  "author": "Daniel Leite de Oliveira <dleitee@gmail.com> (https://github.com/dleitee)",
+  "husky": {
+    "hooks": {
+      "pre-push": "yarn lint"
+    }
+  },
+  "keywords": [],
+  "author": "",
   "license": "ISC",
   "dependencies": {
-    "@zeit/next-css": "^0.2.0",
+    "@zeit/next-css": "^1.0.2-canary.0",
     "@zeit/next-source-maps": "0.0.2",
     "classnames": "^2.2.6",
-    "dotenv": "^6.0.0",
-    "express": "^4.16.3",
-    "fetches": "^0.2.2",
-    "glob": "^7.1.2",
-    "next": "^6.1.1",
-    "next-compose-plugins": "^2.1.1",
+    "compression": "^1.7.3",
+    "creditcard.js": "^2.1.6",
+    "dotenv": "^6.1.0",
+    "express": "^4.16.4",
+    "fetches": "^1.0.1-1.0",
+    "google-map-react": "^1.1.1",
+    "js-cookie": "^2.2.0",
+    "lodash": "^4.17.11",
+    "moment": "^2.22.2",
+    "next": "^8.0.1",
     "next-offline": "^2.9.0",
+    "next-compose-plugins": "^2.1.1",
+    "next-fonts": "^0.10.0",
     "next-optimized-images": "^1.4.1",
-    "next-routes": "^1.4.2",
     "normalize.css": "^8.0.0",
-    "postcss-css-variables": "^0.9.0",
     "prop-types": "^15.6.2",
-    "react": "^16.4.2",
-    "react-dom": "^16.4.2",
-    "react-popover": "^0.5.7",
+    "react": "^16.8.3",
+    "react-content-loader": "^3.2.0",
+    "react-day-picker": "^7.2.4",
+    "react-dom": "^16.8.3",
+    "react-fetches": "^1.0.3",
     "react-portal": "^4.1.5",
```

Comment:
Is `react-portal` required?

Reply:
I will remove the components that its specific to a project